### PR TITLE
Clarify CAP error message, add window border

### PR DESCRIPTION
**Pull request by ahmedalsudani**
@fjvallarino @tejohnso 

### Changes

- Move error messages out to Markdown files
- Clarify CAP error
- Add border around installer window

### CODE REVIEW

File: installer/ui/messages/cap-installed.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+We have detected a previous installation of Rise Chrome App Player. You need to uninstall it before it proceeding with this installation.
```
Comment (tejohnso) [inline installer/ui/messages/cap-installed.md]: typo here "uninstall it before it"

Comment (ahmedalsudani) [inline installer/ui/messages/cap-installed.md]: Oops. Thanks

**Comment by tejohnso** [comment]
Looks good @ahmedalsudani 
Should add a test that confirms messages are being loaded into the global.  

And @ByronDarlison you should check out the updated message for CAP [here](https://github.com/Rise-Vision/rise-launcher-electron/blob/fix/borders-messages/installer/ui/messages/cap-installed.md)

**Comment by ByronDarlison** [comment]
@tejohnso @ahmedalsudani message looks great. Thanks!

**Comment by fjvallarino** [comment]
:+1:

**Comment by ahmedalsudani** [comment]
@tejohnso addressed the issues you raised. Let me know if the test should be moved up a directory (from `test/ui` to `test`) or if the `max-height` change would be problematic

**Comment by tejohnso** [comment]
@ahmedalsudani Looks good to me :+1:

**Pull request merged**